### PR TITLE
Help inline > Add demo and code example

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-help-inline-demo](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-help-inline-demo)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-help-inline-demo](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-help-inline-demo)

File: src/app/docs/help-inline/help-inline-docs.component.html [hunk 1]
```diff
@@ -7,17 +7,23 @@
     The help inline component creates a small button next to a field. When clicked they can trigger an action that displays more information about that field that the user might need information on.
   </sky-docs-demo-page-summary>
 
-  <!-- <sky-docs-demo>
+  <sky-docs-demo>
+    <sky-help-inline>
+    </sky-help-inline>
```
Comment (Blackbaud-AlexKingman) [inline src/app/docs/help-inline/help-inline-docs.component.html:12]: Should this do something when clicked?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-help-inline-demo](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-help-inline-demo)

File: src/app/docs/help-inline/help-inline-docs.component.html (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply src/app/docs/help-inline/help-inline-docs.component.html:12]: Unsure. You want it to show an `alert()`?

Comment (Blackbaud-SteveBrush) [reply src/app/docs/help-inline/help-inline-docs.component.html]: Added an `alert`.

Comment (Blackbaud-AlexKingman) [reply src/app/docs/help-inline/help-inline-docs.component.html]: Yeah. I think your code example already does that. Lets just make them match.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-help-inline-demo](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-help-inline-demo)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**